integration-tests/bouncycastle-fips-jsse/src/main/resources/application.properties

```diff
@@ -16,3 +16,6 @@ quarkus.log.category."org.bouncycastle.jsse".min-level=TRACE
 quarkus.log.category."org.bouncycastle.jsse".level=TRACE
 quarkus.log.file.enable=true
 quarkus.log.file.format=%C - %s%n
+
+#quarkus.native.additional-build-args=--trace-object-instantiation=org.bouncycastle.crypto.fips.FipsSecureRandom
+quarkus.native.additional-build-args=--trace-object-instantiation=java.security.SecureRandom
```

Comment:
FYI you can do `--trace-object-instantiation=java.security.SecureRandom\\,org.bouncycastle.crypto.fips.FipsSecureRandom` to trace both (see second note in  https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/writing-native-applications-tips.adoc#delaying-class-initialization)

Reply:
@zakkak Thanks, will keep it in mind :-), I left the `SecureRandom` tracing as it points out to `JceSecurity`; going forward I'll keep combining several classes when tracing - thanks

Reply:
Let's comment this line maybe?

Reply:
Sure

Reply:
Commented it out and collapsed into a single property configuration as suggested by @zakkak